react/Menu/index.jsx

```diff
@@ -150,7 +189,13 @@ Menu.propTypes = {
   /** `onClick` handler for non disabled items */
   onSelect: PropTypes.func,
   /** `onClick` handler for disabled items */
-  onSelectDisabled: PropTypes.func
+  onSelectDisabled: PropTypes.func,
+  /** itemsStyle : Some global Styles for MenuItems */
```

Comment:
No need to preprend docstrings with the name of the attribute.

"Some" is not necessary. Neither is "global" IMHO.